### DISCUSSION TITLE
missing withMin, withMax, withAvg, withSum, and withExists

### DISCUSSION THREAD

**Discussion by loranger** [filamentphp/filament#1476] [Ideas]
As it's possible to count relationships using
`TextColumn::make('users_count')->counts('users');`

It would be great if the [other aggreation methods](https://laravel.com/docs/8.x/eloquent-relationships#other-aggregate-functions) would be available too
`TextColumn::make('comments_sum_votes')->sums('comments', 'votes');`

And so on…

**Comment by danharrin** [comment]
Sure, I'd be open to a PR to add these.

**Reply by simonbuehler** [reply]
doesn't https://github.com/laravel-filament/filament/pull/2087 fix this?

**Reply by danharrin** [reply]
Yea!
